mu4/notation/internal/measureactionsrepository.cpp

```diff
@@ -24,6 +24,10 @@ using namespace mu::actions;
 ActionList MeasureActionsRepository::actions() const
 {
     ActionList actions;
+    actions.push_back(actionRegister()->action("cut"));
```

Comment:
Maybe better to inherit this class from ElementActionsRepository?

Reply:
The measure is still different from an element